src/qt/optionsmodel.h

```diff
@@ -70,6 +70,7 @@ class OptionsModel : public QAbstractListModel
         SpendZeroConfChange,    // bool
         Listen,                 // bool
         Server,                 // bool
+        EnablePSBTControls,     // bool
```

Comment:
The Clang format check provided the following corrections.

```diff
-        StartAtStartup,         // bool
-        ShowTrayIcon,           // bool
-        MinimizeToTray,         // bool
-        MapPortUPnP,            // bool
-        MapPortNatpmp,          // bool
-        MinimizeOnClose,        // bool
-        ProxyUse,               // bool
-        ProxyIP,                // QString
-        ProxyPort,              // int
-        ProxyUseTor,            // bool
-        ProxyIPTor,             // QString
-        ProxyPortTor,           // int
-        DisplayUnit,            // BitcoinUnits::Unit
-        ThirdPartyTxUrls,       // QString
-        Language,               // QString
+        StartAtStartup,            // bool
+        ShowTrayIcon,              // bool
+        MinimizeToTray,            // bool
+        MapPortUPnP,               // bool
+        MapPortNatpmp,             // bool
+        MinimizeOnClose,           // bool
+        ProxyUse,                  // bool
+        ProxyIP,                   // QString
+        ProxyPort,                 // int
+        ProxyUseTor,               // bool
+        ProxyIPTor,                // QString
+        ProxyPortTor,              // int
+        DisplayUnit,               // BitcoinUnits::Unit
+        ThirdPartyTxUrls,          // QString
+        Language,                  // QString
         UseEmbeddedMonospacedFont, // bool
-        CoinControlFeatures,    // bool
-        SubFeeFromAmount,       // bool
-        ThreadsScriptVerif,     // int
-        Prune,                  // bool
-        PruneSize,              // int
-        DatabaseCache,          // int
-        ExternalSignerPath,     // QString
-        SpendZeroConfChange,    // bool
-        Listen,                 // bool
-        Server,                 // bool
-        EnablePSBTControls,     // bool
+        CoinControlFeatures,       // bool
+        SubFeeFromAmount,          // bool
+        ThreadsScriptVerif,        // int
+        Prune,                     // bool
+        PruneSize,                 // int
+        DatabaseCache,             // int
+        ExternalSignerPath,        // QString
+        SpendZeroConfChange,       // bool
+        Listen,                    // bool
+        Server,                    // bool
+        EnablePSBTControls,        // bool
         OptionIDRowCount,
     };

```

Reply:
This are unrelated changes so I will not include them.

Reply:
Makes Sense 👍